scholarx/archive/2020/mentees.json

```diff
@@ -0,0 +1,426 @@
+        "university": "University of Peradeniya",
+        "linkedin": ""
+    },
+
+    {
+        "name": "Arundika Weerasekera",
+        "image": "arundika.jpg",
+        "mentor": "Nushelle De Silva",
+        "university": "Sri Lanka Institute of Information Technology",
+        "linkedin": "https://www.linkedin.com/in/arundika-weerasekera/"
+    },
+
+    {
+        "name": "Yashodha Warakapitiya",
+        "image": "yashodha.jpg",
+        "mentor": "Nushelle De Silva",
+        "university": "General Sir John Kotelawala Defence University",
+        "linkedin": "https://www.linkedin.com/in/yashodha-warakapitiya-094a4015a/"
+    },
+
+    {
+        "name": "Amaya Dharmasiri",
+        "image": "amaya.jpg",
+        "mentor": "Rukshan Batuwita",
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/amaya-dharmasiri-05558a153/"
+    },
+
+    {
+        "name": "Imesh Ekanayake",
+        "image": "imesh.jpg",
+        "mentor": "Rukshan Batuwita",
+        "university": "University of Peradeniya ",
+        "linkedin": "https://www.linkedin.com/in/imesh-ekanayake-b2b38ba4/"
+    },
+
+    {
+        "name": "Jathurshan Pradeepkumar",
+        "image": "jathurshan.jpg",
+        "mentor": "Rukshan Batuwita",
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/jathurshan-pradeepkumar-319b52104/"
+    },
+
+    {
+        "name": "Yasod Ginige",
+        "image": "yasod.jpg",
+        "mentor": "Rukshan Batuwita",
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/yasod-ginige-42a0011a4/"
+    },
+
+    {
+        "name": "Suwani Gunasekara",
+        "image": "suwani.jpg",
+        "mentor": "Rukshan Batuwita",
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/suwani-gunasekara/"
+    },
+
+    {
+        "name": "Kithmin Wickramasinghe",
+        "image": "kithmin.jpg",
+        "mentor": "Ruwandi Kariyawasam",
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/kithminrw/"
+    },
+
+    {
+        "name": "Amanda Gamage",
+        "image": "amanda.jpg",
+        "mentor": "Ruwandi Kariyawasam",
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/amanda-gamage/"
+    },
+
+    {
+        "name": "Anushiya Thevapalan",
+        "image": "anushiya.jpg",
+        "mentor": "Sajani Amarasiri",
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/anushiya-thevapalan/"
+    },
+
+    {
+        "name": "Miluckshan Jalangan",
+        "image": "maleplaceholder.jpeg",
+        "mentor": "Sajani Amarasiri",
+        "university": "Sri Lanka Institute of Information Technology",
+        "linkedin": ""
+    },
+
+    {
+        "name": "Chamodi Perera",
+        "image": "chamodi.jpg",
+        "mentor": "Sumudu Herath",
```

Comment:
It is Dr. Sumudu Herath. [Apply for all the entries and please check spellings.]